### PR TITLE
Implemented APIs and server

### PR DESCRIPTION
**Pull request by OmkarPh**
Signed-off-by: Omkar Phansopkar <omkarphansopkar@gmail.com>

### CODE REVIEW

File: src/pages/PatientDashboard.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import React from "react"
```
Comment (Aniruddha-Shriwant) [inline src/pages/PatientDashboard.js]: Is it necessary to have this? I don't think current version will give errors if it's not included

**Review by Aniruddha-Shriwant — COMMENTED** [review]
LGTM :)
@Rupesh-2003 will need your review on this 🤯

File: src/pages/PatientDashboard.js (hunk 1 above)
Comment (Aniruddha-Shriwant) [reply src/pages/PatientDashboard.js]: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

> Upgrading to the new transform is completely optional, but it has a few benefits:
>
> - With the new transform, you can use JSX without importing React.
> - ...

Comment (OmkarPh) [reply src/pages/PatientDashboard.js]: right
I installed react 15 / 16 for testing something, had to add this during the process

Comment (OmkarPh) [reply src/pages/PatientDashboard.js]: will remove it

**Review by Rupesh-2003 — APPROVED** [review]

**Pull request merged**